### PR TITLE
Update tokenizer.py

### PR DESCRIPTION
**Pull request by neeravkaushal**
Fix typo. Add unk, cls, sep, and mask tokens to use with standard BERT-based config definitions

## Changelogs

- _enumerate the changes of that PR._

---

_Checklist:_

- [ ] _Add tests to cover the fixed bug(s) or the new introduced feature(s) (if appropriate)._
- [ ] _Update the API documentation if a new function is added, or an existing one is deleted. Eventually consider making a new tutorial for new features._
- [ ] _Write concise and explanatory changelogs below._
- [ ] _If possible, assign one of the following labels to the PR: `feature`, `fix` or `test` (or ask a maintainer to do it for you)._

---

_discussion related to that PR_

### CODE REVIEW

File: safe/tokenizer.py [hunk 1]
```diff
@@ -136,13 +136,33 @@ def bos_token_id(self):
...
+        """Get the unk token id"""
+        return self.tokenizer.token_to_id(self.tokenizer.unk_token)
+    
+    @property
+    def mask_token_id(self):
+        """Get the mask token id"""
+        return self.tokenizer.token_to_id(self.tokenizer.mask_token)
+    
+    @property
+    def cls_token_id(self):
```
Comment (maclandrol) [inline safe/tokenizer.py:158]: For the default tokenizer,  our convention is:

`cls = bos` and `sep = eos`

 So if the cls_token and sep_token are not defined, that's what you should return.
 
 Also remember that you can always use `tokenizer.get_pretrained()`  to get an huggingface PreTrainedTokenizer object

**Review by maclandrol — CHANGES_REQUESTED** [review]
Please see comment for edge case on cls and sep.

**Comment by maclandrol** [comment]
Thanks for the contribution @neeravkaushal, Can you fix the lint (black and ruff ?)

**Comment by neeravkaushal** [comment]
Fixed linting and formatting with `black` and `ruff`

File: safe/tokenizer.py (hunk 1 above)
Comment (neeravkaushal) [reply safe/tokenizer.py:158]: Got it. Thanks :-)

**Pull request merged**